### PR TITLE
remove sql backend toggle

### PR DESCRIPTION
**Pull request by snopoke**
removing this until it's ready for production
@benrudolph @czue 

cc @biyeun @calellowitz

### CODE REVIEW

File: corehq/apps/receiverwrapper/tests/test_submissions.py [hunk 1]
```diff
@@ -13,14 +13,6 @@
...
 
 
-# bit of a hack, but the tests optimize around this flag to run faster
-# so when we actually want to test this functionality we need to set
-# the flag to False explicitly
-from corehq.toggles import USE_SQL_BACKEND, NAMESPACE_DOMAIN
-from toggle.shortcuts import update_toggle_cache, clear_toggle_cache
-
-
-@override_settings(UNIT_TESTING=False)
```
Comment (snopoke) [inline corehq/apps/receiverwrapper/tests/test_submissions.py]: @czue this doesn't seem to be necessary for the tests to pass. It was added by you here: https://github.com/dimagi/commcare-hq/commit/409060d946df2fb89a3d690182978f4cd6ce4353

**Comment by snopoke** [comment]
:white_check_mark:

File: corehq/apps/receiverwrapper/tests/test_submissions.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/receiverwrapper/tests/test_submissions.py]: it seems like different code gets run when this is set to `False` (based on the comment)? having the tests pass might be a misnomer. will let @czue respond

**Comment by benrudolph** [comment]
makes sense to me. will wait for cory to respond to that one question unless you think it's OK

File: corehq/apps/receiverwrapper/tests/test_submissions.py (hunk 1 above)
Comment (czue) [inline corehq/apps/receiverwrapper/tests/test_submissions.py]: it looks like when I added that it was testing the [custom response message](https://github.com/dimagi/commcare-hq/blob/409060d946df2fb89a3d690182978f4cd6ce4353/corehq/apps/receiverwrapper/tests/test_submissions.py), but we killed that functionality entirely so this seems right now.

**Pull request merged**